src/ConfigWidgets/SymbolErrorDialogTest.cpp

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include <QCoreApplication>
+#include <QLabel>
+#include <QPlainTextEdit>
+#include <QPushButton>
+#include <QTest>
+
+#include "ClientData/ModuleData.h"
+#include "ConfigWidgets/SymbolErrorDialog.h"
+#include "GrpcProtos/module.pb.h"
+
+namespace orbit_config_widgets {
+
+orbit_grpc_protos::ModuleInfo CreateModuleInfo(std::string module_path) {
+  orbit_grpc_protos::ModuleInfo module_info;
+  module_info.set_file_path(std::move(module_path));
+  return module_info;
+}
+class SymbolErrorDialogTest : public testing::Test {
+ public:
+  SymbolErrorDialogTest() : module_{CreateModuleInfo(module_path_)}, dialog_(&module_, error_) {}
+
+ protected:
+  void SetUp() override {
+    error_plain_text_edit_ = dialog_.findChild<QPlainTextEdit*>("errorPlainTextEdit");
+    ASSERT_TRUE(error_plain_text_edit_ != nullptr);
+    show_error_button_ = dialog_.findChild<QPushButton*>("showErrorButton");
+    ASSERT_TRUE(show_error_button_ != nullptr);
+  }
```

Comment:
Nit: The gtest documentation encourages to use the constructor instead of the SetUp methods if possible. In this case you could even initialise them in the constructor - similar to module_ and dialog_

Reply:
I did put them in the SetUp method, because in the constructor you cannot use ASSERT